src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
 // write your code here
+const table = document.querySelector('table');
+const rows = table.rows;
+
+for (const row of rows) {
+  const cells = row.cells;
+  let secondColCopy = null;
+
+  if (cells[0].nodeName === 'TH') {
+    secondColCopy = document.createElement('th');
+  } else {
+    secondColCopy = document.createElement('td');
+  }
+
+  secondColCopy.textContent = cells[1].textContent;
+
+  row.insertBefore(secondColCopy, cells[4]);
```

Comment:
Ensure that the fifth column (`cells[4]`) exists in every row before attempting to insert the new column. If a row has fewer than five columns, this line will throw an error. Consider adding a check to ensure `cells[4]` is defined before using it.